tests/test_algorithm.py

```diff
@@ -33,9 +44,9 @@ def test_nnls(rng: np.random.Generator) -> None:
         glass.algorithm.nnls(a.T, b)
 
 
-def test_cov_clip(rng):
+def test_cov_clip(xp: types.ModuleType, urng: UnifiedGenerator) -> None:
     # prepare a random matrix
-    m = rng.random((4, 4))
+    m = xp.asarray(urng.random((4, 4)))
```

Comment:
Is this necessary? I would have hoped that `urng` returns the correct array type

Reply:
Was about to post a comment for this. See https://github.com/glass-dev/glass/pull/610#discussion_r2075129679,